portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,7 +36,7 @@
 /** Servlet that returns comments from datastore */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private String[] imgUrls = {"images/identicon-1.png", "images/identicon-2.png",
+  private static final String[] IMG_URLS = {"images/identicon-1.png", "images/identicon-2.png",
```

Comment:
nit: actually, you can leave the name as `imgUrls`.  The field refers to specific array, and the `final` keyword prevents you from assigning the field to a different array. And `String` in Java is an immutable class, so you can't directly change the  strings contained in the array. But you _can_ change which strings the array contains (e.g., `IMG_URLS[0] = "hello, world";`.

So the Google style guide recommends using lowerCamelCase in this case.